tests/test_settings.py

```diff
@@ -1,3 +1,11 @@
+from django.urls import reverse
+
+import requests_mock
+from django_webtest import WebTest
+
+from open_api_framework.conf.base import LOGIN_URL
```

Comment:
Generally instead of importing directly from settings files, it's best to use `from django.conf import settings`. These are the settings that are actually loaded, so you're guaranteed to have the right settings